src/api/proposal.actions.ts

```diff
@@ -127,20 +120,16 @@ export async function voteOnProposal({
   proposalId: string
 }) {
   if (!Wallet.account?.address) throwError('Wallet not initialized')
-  try {
-    const msg = GroupMsgWithTypeUrl.vote({
-      option,
-      proposalId: Long.fromString(proposalId),
-      voter: Wallet.account.address,
-      exec: 0, // EXEC_UNSPECIFIED
-      metadata: '',
-    })
-    const data = await signAndBroadcast([msg])
-    if (!data) throwError('No data returned from vote')
-    return data
-  } catch (err) {
-    logError(err)
-  }
+  const msg = GroupMsgWithTypeUrl.vote({
+    option,
+    proposalId: Long.fromString(proposalId),
+    voter: Wallet.account.address,
+    exec: 0, // EXEC_UNSPECIFIED
+    metadata: '',
+  })
+  const data = await signAndBroadcast([msg])
+  if (!data) throwError('No data returned from vote')
```

Comment:
should we just use the same msg as in the other `throwError` above? and extract 'No data returned from transaction' into a reusable const string?

```suggestion
  if (!data) throwError('No data returned from transaction')
```

Reply:
https://github.com/regen-network/groups-ui/pull/81/commits/07c4446d20e61c8d5c98cd4d55061590e0372019